### PR TITLE
To-do List: To do interactive list

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request I,
- Have added the functionality **mark task completion** by selecting the corresponding checkbox (or undo it by unchecking the checkbox).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -90,3 +90,4 @@ displayTaskFromStore();
 editFunction();
 
 tasksMenu();
+completedTask();
```
Comment (ahmetbozaci) [inline src/index.js:93]: - [ ] Clear all completed button is not working. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Implement%20a%20function%20for%20the%20%22Clear%20all%20completed%22%20button%20(use%20filter()%20method).)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Sonickmumba  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- I can add and remove todo.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/index.js (hunk 1 above)
Comment (Sonickmumba) [reply src/index.js:93]: @ahmetbozaci 
Thank you so much for the review. The clear all completed button is working from my side. After pressing the clear all completed button, you need to click on the refresh button. All the checked tasks are then cleared.

Comment (ahmetbozaci) [reply src/index.js:93]: @Sonickmumba I can't click on the clear all button. But if it is working on your side, can you send it for review again? The next code reviewer will check.

Comment (rotimiazeez) [reply src/index.js:93]: - Please note that when the clear all completed button is clicked, it only rules over the task without actually clearing them. As suggested by the previous reviewer kindly fix this 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Sonickmumba 👋🏽, 
### Good communication with the reviewer 👍🏽
Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/completedTask.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ const completedTask = () => {
   const clearAllCompleted = document.querySelector('.clear-button');
   clearAllCompleted.addEventListener('click', () => {
     clearCompletedTasks();
+    window.location.reload();
   });
 };
```
Comment (cwaku) [inline src/modules/completedTask.js:29]: [OPTIONAL]
- [ ] Using `window.location.reload` anytime you make changes to the DOM is an efficient way. I recommend you find another way to make changes to the DOM without always reloading the page. This is true for adding and removing tasks. For instance, you could have a function that creates the ToDo list dynamically anytime you add or remove a task. With that, there will be no need to hard reload every time changes are made.

**Review by cwaku — APPROVED** [review]
Hi @Sonickmumba 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/completedTask.js (hunk 2 above)
Comment (Sonickmumba) [reply src/modules/completedTask.js:29]: @cwaku
Noted with thanks.

**Pull request merged**